packages/oc-template-jade-compiler/compile.js

```diff
@@ -0,0 +1,18 @@
+const jade = require('jade-legacy');
+
+module.exports = (options, callback) => {
+  var compiled;
+  try {
+    compiled = jade
+      .compileClient(options.template, {
+        compileDebug: false,
+        name: 't',
+        filename: options.viewPath
+      })
+      .toString()
+      .replace('function t(locals) {', 'function(locals){');
+  } catch (err) {
+    return callback(err);
```

Comment:
I sense that this path is not tested; have you tried to run the coverage?

`--coverage`

😅